### PR TITLE
stm32h7: added minimal stm32h7 exti defs, which share with G0.

### PR DESCRIPTION
**Pull request by vielster**
Separated definitions that did not seem consistent between the "v2" EXTI platforms. Added SYSCFG defs needed for EXTICR settings.

### CODE REVIEW

File: lib/stm32/h7/Makefile [hunk 1]
```diff
@@ -38,12 +38,12 @@ TGT_CFLAGS	+= $(STANDARD_FLAGS)
 ARFLAGS		= rcs
 
 OBJS += dac_common_all.o
+OBJS += exti_common_all.o
 OBJS += flash_common_all.o flash_common_f.o flash_common_f24.o
 OBJS += fmc_common_f47.o
 OBJS += gpio_common_all.o gpio_common_f0234.o
 OBJS += pwr.o rcc.o
 OBJS += rcc_common_all.o
-OBJS += spi_common_all.o spi_common_v2.o
```
Comment (karlp) [inline lib/stm32/h7/Makefile]: this seems out of scope?

**Comment by karlp** [comment]
Looks good apart from the spi thing, was that from rebasing and has slipped in?

**Comment by karlp** [comment]
(I liked the "stm32h7_spi_wtf_st" branch in  your other PR ;)

File: lib/stm32/h7/Makefile (hunk 1 above)
Comment (vielster) [reply lib/stm32/h7/Makefile]: I always have some fun issues cherry-picking across our staging branch to single-feature branches...this should now be untouched.

**Comment by vielster** [comment]
Removed the SPI changes.  Am using this presently to generating rising and falling edge triggered IRQ's on my Nucleo user button :-)  Will be integrating with SPI device driver soon-ish

image

**Comment by karlp** [comment]
Sorry about missing this, my mail server filled it's disk and I didn't get the notifications.  will recheck this tomorrow

**Comment by karlp** [comment]
merged in b1d8a4c5 thanks!
